ironic_prometheus_exporter/app/exporter.py

```diff
@@ -0,0 +1,30 @@
+import configparser
+import logging
+import os
+
+from flask import abort, Flask
```

Comment:
Do we really need `Flask` to expose one endpoint without any routing and any input processing? I guess you could use `werkzeug` or anything similar directly. E.g. for our JSON RPC implementation I used very low level `WebOp` library: https://github.com/openstack/ironic/blob/master/ironic/common/json_rpc/server.py#L102.

Reply:
The only thing that will be doing requests for the Flask Application will be Prometheus, so I think this is fine

Reply:
What I'm trying to say is that Flask is a vast overkill here.

Reply:
Flask probably is overkill, but it's not a huge resource consumer and this is working so it seems fine to me. If we were going to rewrite it, I would suggest doing that in golang to make distribution simpler and to be consistent with other kubernetes components.